tests/app/conftest.py

```diff
@@ -552,16 +553,47 @@ def mmg_provider():
     return ProviderDetails.query.filter_by(identifier="mmg").one()
 
 
+def create_mock_firetext_config(mocker, additional_config=None):
+    config = {
+        "FIRETEXT_URL": "https://example.com/firetext",
+        "FIRETEXT_API_KEY": "foo",
+        "FIRETEXT_INTERNATIONAL_API_KEY": "international",
+        "FROM_NUMBER": "bar",
+    }
+    if additional_config:
+        config.update(additional_config)
+    return mocker.Mock(config=config)
+
+
+def create_mock_firetext_client(mocker, mock_config):
+    client = FiretextClient()
+    statsd_client = mocker.Mock()
+    client.init_app(mock_config, statsd_client)
+    return client
+
+
 @pytest.fixture(scope="function")
 def mock_firetext_client(mocker):
-    client = FiretextClient()
+    mock_config = create_mock_firetext_config(mocker)
+    return create_mock_firetext_client(mocker, mock_config)
+
+
+@pytest.fixture(scope="function")
+def mock_firetext_client_with_receipts(mocker):
+    additional_config = {"FIRETEXT_RECEIPT_URL": "https://www.example.com/notifications/sms/firetext"}
+    mock_config = create_mock_firetext_config(mocker, additional_config)
+    return create_mock_firetext_client(mocker, mock_config)
+
+
+@pytest.fixture(scope="function")
+def mock_mmg_client(mocker):
```

Comment:
Should this be:

```suggestion
def mock_mmg_client_with_receipts(mocker):
```

?